examples/App.jsx

```diff
@@ -484,6 +519,75 @@ export default class App extends Component {
                            </Button>
                         </td>
                   </tr>
+
+
+                  <tr>
+                      <td width="35%">Simple date field</td>
+                      <td width="65%">
+                          <Editable
+                              name="selectedDate"
+                              dataType="date"
+                              mode={"inline"}
+                              title="Enter date"
+                              value="2017-12-31"
+                              handleSubmit={this.handleSubmit}
+                          />
+                          <Panel collapsible expanded={this.state.dateElementInLine} >
+                              <pre> {`<Editable
+                                  name="selectedDate"
+                                  dataType="date"
+                                  mode={"inline"}
+                                  title="Enter date"
+                                  value="2017-12-31"
+                                  handleSubmit={this.handleSubmit}
+                              />
+                              `}</pre>
+                          </Panel>
+                      </td>
+                      <td width="65%">
+                          <Button bsStyle="link" bsSize="xsmall" onClick={ ()=> this.setState({ dateElementInLine: !this.state.dateElementInLine })}>
+                              <i className="fa fa-code" title="show/hide code"></i>
+                          </Button>
+                      </td>
+                  </tr>
+
+
+{/*
```

Comment:
Is radio component in progress?

Reply:
Yeah that one is still WIP and I should have never add it in this PR. Let me take it off from the current PR.